### PR TITLE
Watch stuck jobs

### PR DESCRIPTION
**Pull request by shriramshankar**

### CODE REVIEW

**Comment by shriramshankar** [comment]
The watch stuckJobs won't run until we set the WATCH_STUCK_JOBS_INTERVAL_MILLI_SECONDS config var.

File: config.js [hunk 1]
```diff
@@ -74,6 +73,8 @@ const replicaConfigLabel = 'REPLICAS';
 // an array of read-only data base URLs
 const readReplicas = configUtil.getReadReplicas(pe, replicaConfigLabel);
 
+const WATCH_STUCK_JOBS_INTERVAL_MILLI_SECONDS = Number.MAX_SAFE_INTEGER ||
```
Comment (jgraff2) [inline config.js]: isn't this backwards?

Comment (shriramshankar) [reply config.js]: :) . Will fix it

File: config.js [hunk 2]
```diff
@@ -74,6 +73,8 @@ const replicaConfigLabel = 'REPLICAS';
 // an array of read-only data base URLs
 const readReplicas = configUtil.getReadReplicas(pe, replicaConfigLabel);
 
+const WATCH_STUCK_JOBS_INTERVAL_MILLI_SECONDS =
```
Comment (iamigo) [inline config.js]: "milliseconds" is itself a real word so does not take a `_` in the middle

**Pull request merged**